### PR TITLE
Added new install instructions for pyenv on fish shell

### PR DESCRIPTION
**Pull request by Simba-97**
Added new install instructions for pyenv to the config.fish file.

This pull request resolves issue #423 .

### CODE REVIEW

**Comment by Simba-97** [comment]
Hey, @coolaj86  I believe I have made the required changes, please correct me or request code changes as per review. Thanks!!

**Comment by coolaj86** [comment]
Thanks, but the existing commands should be _replaced_ with new commands, verbatim (do not break the single lines into separate lines).

File: pyenv/install.sh [hunk 1]
```diff
@@ -31,6 +31,9 @@ function __init_pyenv() {
             echo '# added by Webi for pyenv' >> ~/.config/fish/config.fish
             echo 'pyenv init - | source' >> ~/.config/fish/config.fish
```
Comment (coolaj86) [inline pyenv/install.sh:32]: These two existing lines need to be replaced entirely.

The two new lines need to be added verbatim.

```bash
echo 'status is-login; and pyenv init --path | source' >> ~/.config/fish/config.fish
echo 'status is-interactive; and pyenv init - | source' >> ~/.config/fish/config.fish
```

**Review by coolaj86 — CHANGES_REQUESTED** [review]

File: pyenv/install.sh (hunk 1 above)
Comment (coolaj86) [reply pyenv/install.sh:32]: Also, please `git rebase -i main` to squash your changes into a single commit. :)

Comment (Simba-97) [reply pyenv/install.sh]: @coolaj86 the requested changes has been made, please review :)

**Comment by Simba-97** [comment]
@coolaj86 Hey can you please review and let me know if any changes are required?

**Review by coolaj86 — APPROVED** [review]

**Comment by coolaj86** [comment]
This looks correct now, but you didn't sign your commits.

You can use https://webinstall.dev/git-config-gpg to get that set up. It will generate a key for your device and give you the direct URL where you upload to GitHub.

**Comment by Simba-97** [comment]
Thanks, @coolaj86  for the update on the pull request, I have added the signature on the commit and also made the necessary changes in the commit.

**Review by coolaj86 — APPROVED** [review]
Thank you!

If you'd like me to buy you lunch, just drop your Đash address here and I'll send you a tip. :)

**Comment by Simba-97** [comment]
Hey @coolaj86, thanks for your kind gesture, You can tip me at my dash address below. 
Dash address :  XjnaWP7XAJyVJ5pgXscmQC3VSuJSPzgm2F
Thanks once again. Cheers! :)

**Comment by coolaj86** [comment]
@Simba-97 Sent!

**Comment by Simba-97** [comment]
@coolaj86 Thanks once again :)

**Pull request merged**